components/bitly/actions/create-bitlink/create-bitlink.mjs

```diff
@@ -0,0 +1,65 @@
+import { ConfigurationError } from "@pipedream/platform";
+import bitly from "../../bitly.app.mjs";
+import { formatDeepLink } from "../../common/common.utils.mjs";
+
+export default {
+  key: "bitly-create-bitlink",
+  name: "Create a Bitlink",
+  description:
+    "Converts a long url to a Bitlink and sets additional parameters.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    bitly,
+    long_url: {
+      type: "string",
+      description: "URL to shorten",
+    },
+    title: {
+      type: "string",
+      optional: true,
+      description: "Bitlink title",
+    },
+    tags: {
+      type: "string[]",
+      optional: true,
+    },
+    domain: {
+      type: "string",
+      optional: true,
+      description: "Custom domain. e.g. bit.ly",
+    },
+    group_guid: {
+      type: "string",
+      optional: true,
+      description: "Group GUID",
+    },
+    deeplinks: {
+      type: "string[]",
+      optional: true,
+      description: `Provide an object. Each object should represent a row.
+        See documentation: https://dev.bitly.com/api-reference#createFullBitlink
+        Example:
+        \`{
+          "app_id":"com.bitly.app",
+          "app_uri_path": "/store?id=123456",
+          "install_url": "https://play.google.com/store/apps/details?id=com.bitly.app&hl=en_US",
+          "install_type": "promote_install"
+        }\``,
+    },
+  },
+  async run({ $ }) {
+    const { long_url, domain, group_guid, title, tags } = this;
+    const updatedDeepLink = formatDeepLink(this.deeplinks);
+    const payload = { long_url, domain, group_guid, title };
+    tags && tags.length && (payload.tags = tags);
```

Comment:
I'd like to suggest you use an `if` here and in the other cases where you are using `&&` as an `if`. Just to keep better readability and a default code style
Also you can use `tags?.length` instead of `tags && tags.length`

Reply:
Cool. I used tags?.length && (payload.tags = tags); instead